### PR TITLE
ISPN-5685 Filtering doesn't work with sum aggregation

### PR DESCRIPTION
**Pull request by anistor**
jira: https://issues.jboss.org/browse/ISPN-5685
https://issues.jboss.org/browse/ISPN-5666

### CODE REVIEW

File: remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/QueryFacadeImpl.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
  *
  * @author anistor@redhat.com
  * @since 6.0
+ * @private
```
Comment (danberindei) [inline remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/QueryFacadeImpl.java]: Couldn't we just move it to `org.infinispan.query.remote.impl`?

Comment (anistor) [inline remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/QueryFacadeImpl.java]: done

**Comment by danberindei** [comment]
Lots of new failures in the HotRod/integration tests: 
http://ci.infinispan.org/viewLog.html?buildId=29589&buildTypeId=bt9&guest=1

**Comment by anistor** [comment]
Should be ok now.

**Comment by anistor** [comment]
The test failures in CI are not related.

**Comment by gustavonalle** [comment]
pulling

**Comment by gustavonalle** [comment]
Integrated, thanks @anistor !

**Pull request merged**